frontend/src/pages/collab.js

```diff
@@ -84,7 +88,7 @@ function Collab() {
       navigate('/login');
       return;
     }
-    const username = JSON.parse(loggedInUser).username;
+    setUsername(JSON.parse(loggedInUser).username);
```

Comment:
if you set username as state, on the first render the username will still be null which is why i used a local var. maybe u can add another local var and assign it also. cos now the room doesn't contain the usernames of either of the users as u can see in the log,, so it doesnt inform on disconnect.

Reply:
thanks for the feedback, have made the changes